apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.ejs

```diff
@@ -0,0 +1,276 @@
+                  </button>
+                  <button type="button" class="btn btn-sm sr-only js-rubric-item-move-up-button">
+                    Move up
+                  </button>
+                  <button type="button" class="btn btn-sm js-rubric-item-delete text-danger">
+                    <i class="fas fa-trash"></i>
+                    <span class="sr-only">Delete</span>
+                  </button>
+                </td>
+                <td style="max-width: 5rem">
+                  <input type="number" class="form-control js-rubric-item-points" step="any" required>
+                </td>
+                <td>
+                  <input type="text" class="form-control js-rubric-item-description" required maxlength="100">
+                </td>
+                <td>
+                  <label class="js-rubric-item-explanation">
+                    <button type="button" class="btn btn-sm js-rubric-item-long-text-field">
+                      <i class="fas fa-pencil"></i>
+                    </button>
+                  </label>
+                </td>
+                <td>
+                  <label class="js-rubric-item-grader-note">
+                    <button type="button" class="btn btn-sm js-rubric-item-long-text-field">
+                      <i class="fas fa-pencil"></i>
+                    </button>
+                  </label>
+                </td>
+                <td>
+                  New
+                </td>
+              </tr>
+            </template>
+            <% if (variant.params || variant.true_answer || submission?.submitted_answer) { %>
+            <div class="small form-text text-muted">
+              Rubric items may use these entries, which are replaced with the corresponding values for the student variant (click to copy):
+              <ul style="max-height: 7rem; overflow-y: auto;">
+                <% [
+                   ...Object.keys(variant.params || {}).map((key) => ['params', key]),
+                   ...Object.keys(variant.true_answer || {}).map((key) => ['correct_answers', key]),
+                   ...Object.keys(submission?.submitted_answer || {}).map((key) => ['submitted_answers', key]),
+                   ].forEach(([group, key]) => { %>
+                <li><code class="js-copy-on-click"
+                          data-clipboard-text="{{<%= group %>.<%= key %>}}"
+                          >{{<%= group %>.<%= key %>}}</code></li>
+                <% }); %>
+              </ul>
+            </div>
+            <% } %>
+          </div>
+          <hr/>
+          <div class="form-check">
+            <label class="form-check-label">
+              <input class="form-check-input" type="checkbox" value="true"
+                     name="tag_for_manual_grading">
+              Require all graded submissions to be manually graded/reviewed
+            </label>
+          </div>
+        </div>
+        <div class="js-settings-error-alert-placeholder"></div>
```

Comment:
Maybe move this up to be just above the "Add item" button? I think it's better to have the warning close to the controls that are causing the problem, and preferably the button to fix the problem should be just after the warning message.

Reply:
Done in 0b32e15ac. This warning was reusing the same placeholder used for submit errors, these were separated now.